src/main/java/htsjdk/samtools/cram/build/ContainerFactory.java

```diff
@@ -29,7 +29,7 @@
 
 public class ContainerFactory {
     private final SAMFileHeader samFileHeader;
-    private int recordsPerSlice = 10000;
+    private int recordsPerSlice; // default 10000; see CRAMContainerStreamWriter.DEFAULT_RECORDS_PER_SLICE
```

Comment:
The default value here is always overwritten by the `CRAMContainerStreamWriter` default value.  Don't pretend that this line is what sets it.

Reply:
Done - also added a test